bool grasp_mass_0
bool grasp_mass_N
---
bool is_mass0_grasped
bool is_massN_grasped